README.md

```diff
@@ -136,16 +135,19 @@ links: Ember.A([
 ```
 
 If you are using SASS, you must import the styles for the navbar into your styles.
+
 ```scss
 //app.scss
 
-//You can override the styles by setting the following variables:
+// You can override the styles by setting the following variables:
+$c-text-color: #F5F0E1; // cream
```

Comment:
Sweet, I'll move the appropriate variables to this config map then. I'll document it as well.